ops/terraform/services/base/scripts/read-and-decrypt-eyaml.sh

```diff
@@ -17,16 +18,21 @@
 EYAML_FILE="$(basename "$1" | sed 's/\.eyaml//').eyaml"
 SSM_VAULT_PASSWORD_PATH="/bfd/mgmt/jenkins/sensitive/ansible_vault_password"
 
-# temporarily store vault password
-aws ssm get-parameter \
-    --region us-east-1 \
-    --output text \
-    --with-decryption \
-    --query 'Parameter.Value' \
-    --name "$SSM_VAULT_PASSWORD_PATH" > vault.password
 
-# decrypt desired eyaml file to stdout, reformat as JSON, and explicitly cast all values to strings
-ansible-vault decrypt --vault-password-file vault.password "values/${EYAML_FILE}" --output - | yq eval -o=j | jq 'with_entries(.value |= tostring)'
+if test -f "values/${EYAML_FILE}"; then
+    # temporarily store vault password
+    aws ssm get-parameter \
+        --region us-east-1 \
+        --output text \
+        --with-decryption \
+        --query 'Parameter.Value' \
+        --name "$SSM_VAULT_PASSWORD_PATH" > vault.password
+
+    # decrypt desired eyaml file to stdout, reformat as JSON, and explicitly cast all values to strings
+    ansible-vault decrypt --vault-password-file vault.password "values/${EYAML_FILE}" --output - | yq eval -o=j | jq 'with_entries(.value |= tostring)'
+else
+    echo '{}'
```

Comment:
Testing for file existence here and returning `{}` greatly simplifies the downstream `ephemeral.tf` logic.